components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/js/configure-authentication-flow.js

```diff
@@ -78,6 +78,12 @@ function validateAppCreation() {
         return false;
     }
 
+    if (!checkAuthenticators()) {
+        CARBON.showErrorDialog('No authenticators have been configured. Please add atleast one authenticator.',
```

Comment:
Change the message to give the following meaning "You cannot add identifier as the only authenticator. Add more authenticators or add more authentication steps".